starter-code/styles/modules.css

```diff
@@ -25,6 +28,12 @@
   display: block;
 }
 
+@media (min-width: 800px) {
```

Comment:
You have two @media calls for the same min-width. You should only have one for each size. So I would combine these.